README.md

```diff
@@ -15,9 +15,10 @@ If you need help with anything CNCF + Summer of code related, you can file an is
 Communication
 -------------
 
-It's best if you use a public communication channel whenever possible; however, if you need to communicate in private, please feel free to send the admins a note via soc@cncf.io (please use the public channels for any project-related discussion).
+Please reach out to us on the #gsoc channel on the [CNCF slack](https://slack.cncf.io/).
+Please don't use DMs unless strictly necessary.
```

Comment:
@leecalcote @estesp PTA if you are ok with this. I added a line about no DMs because it's just hard to scale that way.

Reply:
Thanks; SGTM